lib/order.rb

```diff
@@ -0,0 +1,18 @@
+class Order
+
+  attr_reader :customer, :menu, :order_details
+
+  def initialize customer, menu, order_details
+    @customer = customer
+    @menu = menu
+    @order_details = order_details
+    order_details.each do |item|
+      fail "Sorry, we don't serve #{item}. Order aborted." unless menu.keys.include? item
```

Comment:
Metrics/LineLength: Line is too long. [89/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)